Ecommerce/app/controllers/pages_controller.rb

```diff
@@ -0,0 +1,15 @@
+class PagesController < ApplicationController
+  def men_fashion
+    @category = Category.find_by(name: 'Male')
```

Comment:
Categories are there so this should be in categories controller why this is in pages controller?